kpops/component_handlers/topic/proxy_wrapper.py

```diff
@@ -44,8 +44,8 @@ def cluster_id(self) -> str:
         bootstrap.servers configuration. Therefore, only one Kafka cluster will be returned.
         :return: The Kafka cluster ID.
         """
-        response = requests.get(url=f"{self._host}/v3/clusters")
-        if response.status_code == requests.status_codes.codes.ok:
+        response = httpx.get(url=f"{self._host}/v3/clusters")
```

Comment:
we should consider creating a httpx Client and setting `self._host` as base url

Reply:
I would do this for the async part, because the client is going to be async too.

Reply:
ok, sounds good